parse.go

```diff
@@ -168,6 +169,14 @@ func WithEnvVarIgnoreCommas(ignore bool) Option {
 	}
 }
 
+// WithIgnoreUndefined tells parse to ignore keys that fail lookup during parse,
+// which would normally throw an error.
+func WithIgnoreUndefined() Option {
+	return func(c *Context) {
+		c.ignoreUndefined = true
```

Comment:
```suggestion
		c.ignoreUndefined = ignore
```

I know it's stupid, but it's consistent with the other bool options.